### PR TITLE
Auto pops up Pledge Modal for first time visitors only

### LINKED ISSUE #1281: Pledge CTA - make the current winning variation default
The [having popup enabled on first page load](https://github.com/mozilla/teach.mozilla.org/issues/1244) variation is winning a lot (~800% improvement) compared to the current default.

Let's make this new winning variation default as Adam suggested
> So I'd recommended committing that winning variation to the repo, pushing it to production, and stopping the current test. Then treat that as your new control to test as many of these further variants as you like / have capacity to build.

And tweak the frequency for the modal so it gets auto popped up once "forever" (or until users or we clear the flag in `localStorage`)

This is a pre-req for https://github.com/mozilla/teach.mozilla.org/issues/1273

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #1281

### CODE REVIEW

**Comment by mmmavis** [comment]
rrrrrrr? @Pomax :dancer: :clap:

File: pages/home.jsx [hunk 1]
```diff
@@ -275,6 +275,12 @@ var HomePage = React.createClass({
     router: React.PropTypes.func.isRequired
   },
   componentDidMount: function() {
+    // auto pops up the Pledge modal if the user is visiting 
+    // the homepage for the first time
+    if (localStorage.getItem("autoOpenPledgeModal") !== "false") {
```
Comment (Pomax) [inline pages/home.jsx]: localStorage is array-ish, so we can say this:
```
if (localStorage["autoOpenPledgeModal"]) {
 ...
}
```

Comment (mmmavis) [inline pages/home.jsx]: I think I'm gonna rename the var to "disableAutoOpenPledgeModal" and do
```js
if (!localStorage.getItem("disableAutoOpenPledgeModal")) {
  this.handlePledgeBtnClick();
  localStorage.setItem("disableAutoOpenPledgeModal", "true");
}
```

this way the logic looks more intuitive.  agree? @Pomax

Comment (Pomax) [inline pages/home.jsx]: good idea. Worth caching the string then, too?
```
var disableModal = "disableAutoPledgeModal";
if (!localStorage[disableModal]) {
  this.handlePledgeBtnClick();
  localStorage.setItem(disableModal, "disabled");
}
```

**Comment by mmmavis** [comment]
waiting for Travis to finish then i'll merge thisssss

**Comment by Pomax** [comment]
woop

**Comment by mmmavis** [comment]
:tada:

**Pull request merged**